app/config/GuiceModule.scala

```diff
@@ -42,4 +48,15 @@ class GuiceModule(
     bind(classOf[Clock]).toInstance(Clock.systemUTC())
   }
 
+  protected def bindBoolean(path: String, name: String = ""): Unit =
+    bindConstant()
+      .annotatedWith(named(resolveAnnotationName(path, name)))
+      .to(Try(configuration.get[String](path).toBoolean).toOption.getOrElse(configException(path))) //We need to parse as string, due to the process of adding in from app-config-<env> it is seen as a string
+
+  private def resolveAnnotationName(path: String, name: String): String = name match {
```

Comment:
I don't know why we bother with `resolveAnnotationName`. Isn't it better to always provide a name?